### PR TITLE
Fix missing or false description strings

### PR DESCRIPTION
**Pull request by tobolar**
This concerns description strings of 
- classes
- components
- parameters

(Detected by style check.)

### CODE REVIEW

File: ThermofluidStream/FlowControl/MCV.mo [hunk 1]
```diff
@@ -25,7 +25,7 @@ model MCV "Massflow and volume control valve"
 
 
   parameter Mode mode = Mode.mass_flow "Valve mode";
-  parameter Boolean setpointFromInput = false "Enable desired massFlow input";
+  parameter Boolean setpointFromInput = false "= true, if desired setpoint_var input enabled";
```
Comment (nieweber) [inline ThermofluidStream/FlowControl/MCV.mo:28]:
```suggestion
  parameter Boolean setpointFromInput = false "= true, if desired flow rate is set via setpoint_var input";
```

File: ThermofluidStream/HeatExchangers/Tests/FlowRes.mo [hunk 2]
```diff
@@ -9,7 +9,7 @@ model FlowRes
 
   extends Modelica.Icons.Example;
 
-  parameter Integer nCells = 10;
+  parameter Integer nCells = 10 "Number of evaporators' discretization elements";
```
Comment (nieweber) [inline ThermofluidStream/HeatExchangers/Tests/FlowRes.mo:12]:
```suggestion
  parameter Integer nCells = 10 "Number of discretization elements in heat exchangers";
```

File: ThermofluidStream/Undirected/FlowControl/MCV.mo [hunk 3]
```diff
@@ -23,7 +23,7 @@ model MCV "Massflow control valve"
         origin={0,-80})));
 
   parameter Mode mode = Mode.mass_flow "Valve mode";
-  parameter Boolean setpointFromInput = false "Enable desired massFlow input";
+  parameter Boolean setpointFromInput = false "= true, if desired setpoint_var input enabled";
```
Comment (nieweber) [inline ThermofluidStream/Undirected/FlowControl/MCV.mo:26]:
```suggestion
  parameter Boolean setpointFromInput = false "= true, if desired flow rate is set via setpoint_var input";
```

**Review by nieweber — CHANGES_REQUESTED** [review]

**Comment by tobolar** [comment]
@nieweber I have commited your suggestions.

**Pull request merged**